requirements.txt

```diff
@@ -2,8 +2,8 @@
 tf-nightly==2.15.0.dev20231009  # Pin a working nightly until rc0.
 
 # Torch.
-torch>=2.0.1
-torchvision>=0.15.1
+torch>=2.1.0
```

Comment:
@grasskin - FYI.  I remember Gabriel wanting to keep the requirements as `torch 2.0.1`.  So wanted him to take a look or be in the loop.

Reply:
thanks @sampathweb, @grasskin, let me know if we have a good reason to stay at 2.0.1. I'd like to update to 2.1 if possible since it has a bunch of fixes (especially to torch.compile)